src/storage/test/AdminTaskManagerTest.cpp

```diff
@@ -33,6 +33,8 @@ namespace storage {
  *      6.2 cancel_a_task_while_some_sub_task_running
  * */
 
+// using ResultCode = nebula::kvstore::ResultCode;
+using ResultCode = cpp2::ErrorCode;
```

Comment:
not sure it's a good idea maybe make me confuse with kvstore ResultCode

Reply:
yes, a would like to fix it in another pr